src/components/Buttons/BasicButton/BasicButton.jsx

```diff
@@ -0,0 +1,13 @@
+import { BasicButtonLayout } from '@/components/Buttons/BasicButton/BasicButtonStyled';
+
+const BasicButton = props => {
```

Comment:
```suggestion
const BasicButton = ({ text, handleClick, variant, sx }) => {
```

Reply:
thank you, did know it's possible to decompose in such way.